### PR TITLE
Covid tracking us

### PR DESCRIPTION
**Pull request by wh1210**

### CODE REVIEW

File: scripts/covid_tracking_project/historic_us_data/COVIDTracking_US.tmcf [hunk 1]
```diff
@@ -0,0 +1,96 @@
+
+Node: E:COVIDTracking_US->E0
+typeOf: dcs:StatVarObservation
+variableMeasured: dcs:CumulativeCount_MedicalTest_COVID_19
+measurementMethod: dcs:CovidTrackingProject
+observationAbout: dcs:country/USA
```
Comment (tjann) [inline scripts/covid_tracking_project/historic_us_data/COVIDTracking_US.tmcf:6]: nit: country/USA is not schema, so we prefer dcid:

File: scripts/covid_tracking_project/historic_us_data/COVIDTracking_US_StatisticalVariables.mcf [hunk 2]
```diff
@@ -0,0 +1,94 @@
+Node: dcid:CumulativeCount_MedicalTest_COVID_19
```
Comment (tjann) [inline scripts/covid_tracking_project/historic_us_data/COVIDTracking_US_StatisticalVariables.mcf:1]: Remember that StatVars only need to be checked in once. If these are all the same as States, then we can just reuse.

File: scripts/covid_tracking_project/historic_us_data/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+This directory contains artifacts required for importing
+The COVID Tracking Project's Historic US Data
+into Data Commons, along with scripts used to generate these artifacts.
+
+## Artifacts:
+
+- [COVIDTracking_US.csv](COVIDTracking_US.csv): the cleaned CSV.
+- [COVIDTracking_US.tmcf](COVIDTracking_US.tmcf): the mapping file (Template MCF).
+- [COVIDTracking_US_StatisticalVariables.mcf](COVIDTracking_US_StatisticalVariables.mcf):
```
Comment (tjann) [inline scripts/covid_tracking_project/historic_us_data/README.md:11]: Can remove.

File: scripts/covid_tracking_project/historic_us_data/README.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+  the new StatisticalVariables defined for this dataset.
+
+## Generating Artifacts:
+
+`COVIDTracking_US_StatisticalVariable.mcf` was handwritten.
```
Comment (tjann) [inline scripts/covid_tracking_project/historic_us_data/README.md:16]: rm

File: scripts/covid_tracking_project/historic_us_data/preprocess_csv.py [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+                  'CumulativeCount_MedicalConditionIncident_COVID_19_PatientRecovered',
+                  'CumulativeCount_MedicalConditionIncident_COVID_19_PatientDeceased',
+                  'Count_MedicalConditionIncident_COVID_19_PatientHospitalized',
+                  'CumulativeCount_MedicalConditionIncident_COVID_19_PatientHospitalized',
+                  'Count_MedicalConditionIncident_COVID_19_PatientInICU',
+                  'CumulativeCount_MedicalConditionIncident_COVID_19_PatientInICU',
+                  'Count_MedicalConditionIncident_COVID_19_PatientOnVentilator',
+                  'CumulativeCount_MedicalConditionIncident_COVID_19_PatientOnVentilator'
+                 ]
+with open('COVIDTracking_US.csv', 'w', newline='') as f_out:
```
Comment (tjann) [inline scripts/covid_tracking_project/historic_us_data/preprocess_csv.py:19]: I have an idea. For a subsequent PR, could we modularize State and US code, such that:

- During CSV cleaning phase, add a column 'GeoId' for US where all values are "country/USA"

- A catch is the typeOf for E0 is hardcoded as State. We can table this discussion for now, just something to mull over if you want.

Comment (wh1210) [reply scripts/covid_tracking_project/historic_us_data/preprocess_csv.py:19]: Yep! Will have a future version to combine these.

File: scripts/covid_tracking_project/historic_us_data/README.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+
+To generate `COVIDTracking_US.tmcf` and `COVIDTracking_US.csv`, run:
+
+```bash
+python3 preprocess_csv.py
+```
+
+#### Note:
+
+`COVIDTracking_US.mcf` is actually the same with `COVIDTracking_States.mcf`, so we don't need to actually check it.
```
Comment (tjann) [inline scripts/covid_tracking_project/historic_us_data/README.md:15]: Do these files exist?

Comment (wh1210) [reply scripts/covid_tracking_project/historic_us_data/README.md:15]: Sorry for missing StatVars after the file name.

File: scripts/covid_tracking_project/historic_us_data/README.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+
+To generate `COVIDTracking_US.tmcf` and `COVIDTracking_US.csv`, run:
+
+```bash
+python3 preprocess_csv.py
+```
+
+#### Note:
+
+`COVIDTracking_US_StatisticalVariables.mcf` is actually the same as `COVIDTracking_States_StatisticalVariables.mcf` in /historic_state_data/ folder, so we don't need to check it here.
```
Comment (tjann) [inline scripts/covid_tracking_project/historic_us_data/README.md:15]: Could we just remove the file?

File: scripts/covid_tracking_project/historic_us_data/config.txt [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+    provenance_description: "COVID-19 related data for US from The COVID Tracking Project."
+    # Path to the resolved MCF.
+    mcf_url: "gs://resolved_mcf/external_tables/covid_tracking_us/nodes.mcf-00000-of-00001"
+    # First date of imported data.
+    start_date: "2020-01-22"
+    # Last date of imported data.
+    end_date: "2020-06-22"
+    # Source data release schedule.
+    update_frequency: "P1D"
+    next_release_date: "2020-06-23"
```
Comment (tjann) [inline scripts/covid_tracking_project/historic_us_data/config.txt:13]: Can remove end_date and next_release_date for now

Comment (wh1210) [reply scripts/covid_tracking_project/historic_us_data/config.txt]: Fixed~

File: scripts/covid_tracking_project/historic_us_data/README.md (hunk 7 above)
Comment (wh1210) [reply scripts/covid_tracking_project/historic_us_data/README.md]: Deleted

Comment (tjann) [reply scripts/covid_tracking_project/historic_us_data/README.md]: I meant delete the COVIDTracking_US_StatisticalVariables.mcf not the README 😬 Sorry for the confusion

Comment (wh1210) [reply scripts/covid_tracking_project/historic_us_data/README.md]: Hahaha, no worry~ Recovered

**Pull request merged**